migration/migrating_3_4/configuring-replication-repository.adoc

```diff
@@ -1,22 +1,20 @@
-[id='configuring-replication-repository_{context}']
+[id='configuring-replication-repository_migrating-3-4']
```

Comment:
I can't speak for sure because I'm not familiar with nesting assemblies, but I think that this _{context} should actually stay here. It's only when you're actually using an xref to reference that it should be resolved out.